### DISCUSSION TITLE
Updates von Downloads?

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#192] [Q&A] [answered]
Habe seit heute diese Fehlermeldung. ??

App funktioniert aber eigentlich normal. In Playstore gibt es keine Updates für Cruiser. Gibt es eine Idee was der Grund sein könnte.

<img src="https://github.com/devemux86/cruiser/assets/124886589/e48ec94b-4421-4fa9-a619-670620d0cba7" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/124886589/857e08d2-a81b-4ff2-b3f9-ebd27569dceb" width="50%"/>

**Comment by devemux86** [comment, marked as answer]
@1maxi2 

It means that there are updates to your downloads (Maps, POI, GraphHopper).
It also appears in the various menus. See also the pictures in this thread:
Maybe we need a better message or no message?

Es bedeutet, dass Aktualisierungen Ihrer Downloads (Karten, POI, GraphHopper) vorliegen.
Es erscheint auch in den verschiedenen Menüs. Siehe auch die Bilder in diesem Thema:
Vielleicht brauchen wir eine bessere Botschaft oder keine Botschaft?

- https://github.com/devemux86/cruiser/discussions/86#discussioncomment-6023184

**Reply by 1maxi2** [reply]
Okay es sind update notwendig. Wie kann ich die update der Karten durchführen ?
![Screenshot_20230530_191945_Cruiser](https://github.com/devemux86/cruiser/assets/124886589/77f6f0db-6173-49f7-b9a2-4f74f60d81f3)
![Uploading Screenshot_20230530_191937_Cruiser.jpg…]()

**Reply by devemux86** [reply]
@1maxi2 

> Wie kann ich die update der Karten durchführen ?

The updates are located on the 3rd screen of the download manager.

Die Updates befinden sich auf dem 3. Bildschirm des Download-Managers.

**Reply by devemux86** [reply]
Screens in the Download manager:
1. Downloads
2. Local files
3. Updates

Bildschirme im Download-Manager:
1. Herunterladen
2. Lokale Dateien
3. Aktualisierungen

**Reply by 1maxi2** [reply]
Sorry finde die Aktualisierung nicht 🤔 Werde mal eine Nacht darüber schlafen vielleicht hilft es....

**Reply by devemux86** [reply]
@1maxi2 

Open the download manager for the maps and press the 3rd screen with the clock.

Öffnen Sie den Karten-Download-Manager und drücken Sie auf den dritten Bildschirm mit der Uhr.

<img src="https://github.com/devemux86/cruiser/assets/3484020/01589bbc-38a9-4cf7-98af-5328d384c88a" width="50%"/>

**Reply by 1maxi2** [reply]
So jetzt hat die Aktualisierung funktioniert. Allerdings hat ausgerechnet die Austria map " ewig" gedauert trotz relativ schnellen Internet.
Wird da nur eine Aktualisierung heruntergeladen oder jedesmal die ganze map (~ 480 MB  ) ?
Der Beitrag kann dann gelöscht werden da ja die Cruiser app keinen Fehler hat sondern da es nur ein Anwenderfehler von mir war 🫣

**Reply by devemux86** [reply]
> Wird da nur eine Aktualisierung heruntergeladen

If we click on multiple updates, all of them will be downloaded.
The downloads are done by Android, the app just tells Android which files to download.

Wenn wir auf mehrere Updates klicken, werden alle heruntergeladen.
Die Downloads werden von Android durchgeführt, die App teilt Android lediglich mit, welche Dateien heruntergeladen werden sollen.

**Reply by 0709wiwiwi** [reply]
> Allerdings hat ausgerechnet die Austria map " ewig" gedauert trotz relativ schnellen Internet

In noticed exactly the same. Did not try again today (mobile now)
